src/woo_publications/metadata/models.py

```diff
@@ -58,6 +61,43 @@ class InformationCategory(OrderedModel):
         max_length=15,
         default=InformationCategoryOrigins.custom_entry,
     )
+    # Retention fields:
+    bron_bewaartermijn = models.CharField(
+        _("retention source"),
+        help_text=_(
+            "The source of the retention (example: Selectielijst gemeenten 2020)."
+        ),
+        max_length=255,
+        null=True,
+        blank=False,
+    )
```

Comment:
```suggestion
    bron_bewaartermijn = models.CharField(
        _("retention policy source"),
        help_text=_(
            "The source of the retention policy (example: Selectielijst gemeenten 2020)."
        ),
        max_length=255,
        blank=False,
        default=""
    )
```
* I think "retention policy" is a better translation for "bewaartermijn".
* Never use `null=True` for `CharField`/`TextField`, it's confusing what the empty value is.
* Because it's `blank=True`, I've added an explicit `default=""` which will work at the DB level, but in the admin the field remains required. This is also the default of Django, but it shows it was a deliberate action.